test/suites/basic.sh

```diff
@@ -77,7 +77,7 @@ test_basic_usage() {
   lxc profile create priv
   lxc profile set priv security.privileged true
   lxc init testimage barpriv -p default -p priv
-  lxc publish barpriv --alias=foo-image prop1=val1
```

Comment:
It works either with:

    lxc publish --alias=foo-image barpiv prov1=val1

or:

    lxc publish busybox prov1=val1 --alias=busybox